### PR TITLE
Demo page to gh-pages

### LINKED ISSUE #45: Create a demo page
We need to create a demo page to show and use as real test.

Pushing directly to github pages is good.

### PR DESCRIPTION
**Pull request by EL-shadow**
Demo page for testing in browser resolve #45 
 from https://el-shadow.github.io/Detect.js/
to https://darcyclarke.github.io/Detect.js/

### CODE REVIEW

**Comment by AdrianoCahete** [comment]
I think you can use the lib from master branch, but i don't know how. I'll search for, it's easy to maintain udapted.

**Comment by julmot** [comment]
With RawGit

**Comment by EL-shadow** [comment]
fixed

File: index.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Detect JS</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <script src="https://rawgit.com/darcyclarke/Detect.js/master/detect.js"></script>
```
Comment (AdrianoCahete) [inline index.html]: In RawGit, they tell us to use cdn endpoint to create cache. But the cache don't expire, so we need to point not to master file, but the release/tag file: `https://cdn.rawgit.com/darcyclarke/Detect.js/v2.2.2/detect.js`

**Review by AdrianoCahete — CHANGES_REQUESTED** [review]

**Comment by AdrianoCahete** [comment]
I didn't test, just point out the file change. Please, test before.

**Comment by EL-shadow** [comment]
I update rawgit link to cdn tagged version and test page  - correctly work.

**Review by AdrianoCahete — APPROVED** [review]

**Comment by AdrianoCahete** [comment]
I'll try to test and merge to master this week.

**Pull request merged**